modules/theme/supported_themes.nix

```diff
@@ -92,4 +92,40 @@
     '';
     styles = ["dark" "light"];
   };
+
+  gruvbox = {
+    setup = {
+      style ? "dark",
+      transparent ? false,
+    }: ''
+      -- Gruvbox theme
+      require("gruvbox").setup({
+        terminal_colors = true, -- add neovim terminal colors
+        undercurl = true,
+        underline = true,
+        bold = true,
+        italic = {
+        strings = true,
+        emphasis = true,
+        comments = true,
+        operators = false,
+        folds = true,
+        },
+        strikethrough = true,
+        invert_selection = false,
+        invert_signs = false,
+        invert_tabline = false,
+        invert_intend_guides = false,
+        inverse = true,
+        contrast = "",
+        palette_overrides = {},
+        overrides = {},
+        dim_inactive = false,
+        transparent_mode = false,
```

Comment:
Is there a reason we are hardcoding the transparency option?

You could do `transparent_mode = ${lib.boolToString transparent}` here, unless you decided to disable it foro a reason.

Reply:
Just a minor slip up on my part. I didn't test, nor have I ever used, transparency, so I just didn't think about it. I changed it now, to what you proposed.

I also added an entry to the 0.6 release-notes, as mentioned by @FrothyMarrow.